addons/website_sale/static/src/js/location_selector/map/map.xml

```diff
@@ -1,30 +1,84 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="website_sale.locationSelector.map">
-        <div class="w-100 h-100" style="width: 30rem! important; height: 30rem !important;" t-ref="mapContainer"></div>
+        <div class="d-flex flex-column h-100 w-100">
+            <div class="w-100 flex-grow-1" t-ref="mapContainer"/>
+
+            <!-- Desktop infos -->
+            <div
+                t-if="selectedLocation"
+                class="d-none d-md-flex justify-content-between gap-1 gap-lg-4 flex-column flex-lg-row align-items-start p-4"
+            >
+                <div class="d-flex gap-3">
+                    <strong
+                        class="fs-5 fw-bold"
+                        t-out="this.props.locations.indexOf(selectedLocation) + 1"
+                    />
+                    <div class="d-flex flex-column">
+                        <strong class="fs-5 fw-bold" t-out="selectedLocation.pick_up_point_name"/>
+                        <span t-out="selectedLocation.pick_up_point_address"/>
+                        <!-- TODO VCR postal code is hardcoded before the adress. I might not suits some l10n -->
+                        <span t-out="selectedLocation.postal_code + ' ' + selectedLocation.pick_up_point_town"/>
+
+                        <button
+                            type="button"
+                            class="btn btn-primary mt-3"
+                            t-att-disabled="!this.props.selectedLocationId"
+                            t-on-click="this.props.validateSelection">
+                                Choose this location
+                        </button>
+                    </div>
+                </div>
+                <hr class="d-lg-none"/>
+
+                <!-- Schedule -->
+                <LocationSchedule
+                    openingHours="selectedLocation.opening_hours"
+                    wrapClass="'flex-grow-1 flex-lg-grow-0'"
+                />
```

Comment:
Isn't it possible to use the Location component here ?
```suggestion
               <Location
                        id="selectedLocation.id.toString()"
                        name="selectedLocation.pick_up_point_name"
                        address="selectedLocation.pick_up_point_address"
                        number="location_index + 1"
                        openingHours="selectedLocation.opening_hours"
                        isSelected="this.props.selectedLocationId === selectedLocation.id.toString()"
                        setSelectedLocation="this.props.setSelectedLocation"
                    />
```

Reply:
The customizations between the location structure in the list view and in the map view on **desktop** are too different to work on the same component.

But between the list view and the map view on **mobile**, its almost the same.

So I adapted the Location component to work in both the list view and the map view on mobile.
Only map view on desktop uses its own customizations.

I've created a new commit so you can easily check and decide whether you want to keep it or not 🙂 .